docker/bin/bootstrap.sh

```diff
@@ -169,13 +169,18 @@ install() {
 	USER="admin"
 	PASSWORD="admin"
 
+	if [[ "$MYSQL_ROOT_PASSWORD" == "" ]]; then
```

Comment:
```suggestion
	if [[ "$MYSQL_DB_USER" == "" ]]; then
```

As it is not used as the password